### PR TITLE
Filebeat sophos xg bring back the system health pipeline. #27826

### LINKED ISSUE #27826: Filebeat Sophos module pipeline.yml references to missing system health pipeline
Version: 7.14.1

In the pipeline.yml for the sophos xg module it shows me this:

```yaml
- pipeline:
    name: '{< IngestPipeline "systemhealth" >}'
    if: "ctx.sophos?.xg?.log_type == 'System Health'"
```

However, there is no `systemhealth.yml`

```bash
➜  ingest ls -ls
total 176
16 -rw-r--r--@ 1 philippkahr  staff   6126 Aug 26 11:12 antispam.yml
24 -rw-r--r--@ 1 philippkahr  staff   9813 Aug 26 11:12 antivirus.yml
16 -rw-r--r--@ 1 philippkahr  staff   6956 Aug 26 11:12 atp.yml
16 -rw-r--r--@ 1 philippkahr  staff   7574 Aug 26 11:12 cfilter.yml
16 -rw-r--r--@ 1 philippkahr  staff   7335 Aug 26 11:12 event.yml
24 -rw-r--r--@ 1 philippkahr  staff  11974 Aug 26 11:12 firewall.yml
16 -rw-r--r--@ 1 philippkahr  staff   5961 Aug 26 11:12 idp.yml
16 -rw-r--r--@ 1 philippkahr  staff   7136 Aug 26 11:12 pipeline.yml
 8 -rw-r--r--@ 1 philippkahr  staff   3159 Aug 26 11:12 sandstorm.yml
16 -rw-r--r--@ 1 philippkahr  staff   7977 Aug 26 11:12 waf.yml
 8 -rw-r--r--@ 1 philippkahr  staff    446 Aug 26 11:12 wifi.yml
 ```

As far as I can tell, it was removed in this [commit](https://github.com/elastic/beats/commit/34ee591fc8fa76f03ccd0809b68c3392582ecd04)

When I use that module, I see `error.message` populated with `Pipeline processor configured for non existent pipeline ....-xg-systemhealth`.

### PR DESCRIPTION
**Pull request by philippkahr**
## What does this PR do?

Brings back the system health pipeline

## Why is it important?

We are supporting sophos xg module, however our pipeline.yml references to a non existing system health pipeline. This was first removed (as far as me digging through the git history in this [commit](https://github.com/elastic/beats/commit/34ee591fc8fa76f03ccd0809b68c3392582ecd04))

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] ~~I have commented my code, particularly in hard-to-understand areas~~
- [ ] ~~I have made corresponding changes to the documentation~~
- [ ] ~~I have made corresponding change to the default configuration files~~
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Related issues

- Closes #27826

### CODE REVIEW

**Comment by philippkahr** [comment]
This is still a work in progress, I just discovered that the original pipeline was not matched to ECS properly. So gonna take some time to resolve this.

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -312,6 +312,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Store offset in `log.offset` field of events from the filestream input. {pull}27688[27688]
 - Fix `httpjson` input rate limit processing and documentation. {pull}[]
 - Update Filebeat compatibility function to remove processor description field on ES < 7.9.0 {pull}27774[27774]
+- Fix Filebeat Sophos XG module. It referred to a non existing pipeline for systemhealth. {pull}27826[27826]
```
Comment (adriansr) [inline CHANGELOG.next.asciidoc:315]: With this wording it's unclear what the fix was, someone can wonder if just the reference was removed.

```suggestion
- sophos/xg fileset: Added missing pipeline for System Health logs. {pull}27826[27826]
```

**Review by adriansr — DISMISSED** [review]
Do you have some samples of these System Health logs so that we can add them to the test suite?

**Review by P1llus — COMMENTED** [review]
Added two small comments

**Review by P1llus — APPROVED** [review]

**Pull request merged**